### DISCUSSION TITLE
Docs: Argocd to Flux2 using "app-of-apps" pattern.

### DISCUSSION THREAD

**Discussion by nwmcsween** [fluxcd/flux2#3018] [General] [answered]
ArgoCD generally uses an "app-of-apps" pattern to do helm releases, I tried to translate this to flux2 a while back but ran into blocking issues, if possible could flux2 provide docs on creating a helm centric repo instead of a kustomize one?

From what I remember the complexity got out of hand doing an "app-of-apps" pattern using the HelmRepository abstractions.

**Comment by stefanprodan** [comment]
Please see this example repo that uses Helm releases for apps: https://github.com/fluxcd/flux2-kustomize-helm-example

**Comment by nwmcsween** [comment]
The problem with that is it depends on kustomize to "template" and not
helm. I think the problems I ran into were the abstractions provided by
flux2 and that I don't think subcharts get reconciled correctly(?) made a
helm app-of-apps unworkable.

**Reply by stefanprodan** [reply]
Please open an issue in helm-controller if sub charts don't work for you.

**Comment by nwmcsween** [comment, marked as answer]
I will remake the fluxcd repo again and open discussions/issues as I come
across them.
